### ISSUE TITLE
Amavis hostname is set to a wrong value

### ISSUE THREAD

**Issue by jeroenrnl** [technicalguru/docker-mailserver-amavis#20]
A "virus blocked" notification fails with `501 5.5.4 Bad ENVID parameter syntax`

This is due to
**`05-node-id`**:
```
$myhostname = `uname -a`.'domain.com';
```

this should be `uname -n`

**Comment by jeroenrnl** [comment]
The `uname -n` command also adds a newline that postfix doesn't find amusing :smile:. I settled on:
```
    use strict;

    my $myhost;
    # $myhostname is used by amavisd-new for node identification, and it is
    # important to get it right (e.g. for ESMTP EHLO, loop detection, and so on).

    chomp($myhost = `uname -n`);
    $myhostname = $myhost.'.__AV_MYDOMAIN__';

    1;  # ensure a defined return
```

**Comment by technicalguru** [comment]
Thanks for the report. Will add it (not before weekend though).

**Comment by technicalguru** [comment]
Released in fix 2.11.1.6

**Comment by technicalguru** [comment]
@jeroenrnl - Thank you for contribution
